README.md

```diff
@@ -27,71 +25,121 @@ Lance makes machine learning workflows with ML data easy (images, videos, point
-```
+# Sample 100 query vectors
+tbl = dataset.to_table()
+sample = duckdb.query("SELECT vector FROM tbl USING SAMPLE 100").to_df()
+query_vectors = np.array([np.array(x) for x in sample.vector])
 
-For more details, read our [documentation](https://eto-ai.github.io/lance/notebooks/02_creating_lance_datasets.html).
+# Get nearest neighbors for all of them
+rs = [dataset.to_table(nearest={"column": "vector", 
+                                "k": 10, 
+                                "q": query_vectors[i, :]}) 
+      for i in range(query_vectors.shape[0])]
+```
 
-## Important directories
+## Directory structure
 
-| Directory                                  | Description                            |
-|--------------------------------------------|----------------------------------------|
-| [cpp](./cpp)                               | Core Lance Format |
-| [python](./python)                         | Python SDK (Pylance)                |
-| [notebooks](./python/notebooks)            | Jupyter Notebooks                      |
-| [duckdb extension](./integration/duckdb)   | Lance Duckdb extension                 |
+| Directory          | Description              |
+|--------------------|--------------------------|
+| [rust](./rust)     | Core Rust implementation |
+| [python](./python) | Python bindings (pyo3)   |
+| [docs](./docs)     | Documentation source     |
 
 ## What makes Lance different
 
-Here we will highlight a few aspects of Lance’s design. For more details, see the full [Lance design document](https://docs.google.com/document/d/1kknVcqRK65YqGkKASuQ40apr2A2DyK0Qtx5nhCPCdqQ/edit).
+Here we will highlight a few aspects of Lance’s design. For more details, see the full [Lance design document](https://eto-ai.github.io/lance/format.html).
 
 **Encodings**: to achieve both fast columnar scan and sub-linear point queries, Lance uses custom encodings and layouts.
 
 **Nested fields**: Lance stores each subfield as a separate column to support efficient filters like “find images where detected objects include cats”.
 
-**Versioning / updates** (ROADMAP): a Manifest can be used to record snapshots. Updates are supported via write-ahead logs.
+**Versioning**: a Manifest can be used to record snapshots. Currently we support creating new versions automatically via appends, overwrites, and index creation 
+
+**Fast updates** (ROADMAP): Updates will be supported via write-ahead logs.
+
+**Vector index**: Vector index for similarity search over embedding space
 
-**Secondary Indices** (ROADMAP):
-  - Vector index for similarity search over embedding space
+**Rich secondary indices** (ROADMAP): 
   - Inverted index for fuzzy search over many label / annotation fields
 
 ## Benchmarks
 
+### Vector search
+
+We used the sift dataset to benchmark our results with 1M vectors of 128D
+
+1. For 100 randomly sampled query vectors, we get <1ms average response time (on a 2023 m2 macbook air)
+
+![img.png](img.png)
```

Comment:
In the image, change the dataset file name to `sift_1m.lance`?

Reply:
i think in the tarball the directory is vec_data.lance, so that's what ppl will see by default

Reply:
oh, lemme make a new tarball, this is the SIFT dataset, right